tests/autoclassify/test_classify_failures.py

```diff
@@ -34,7 +34,7 @@ def autoclassify(jm, job, test_failure_lines, matchers, status="testfailed"):
 
     register_matchers(*matchers)
 
-    call_command('autoclassify', jm.project, job.guid)
+    call_command('autoclassify', str(job.id))
```

Comment:
I'm guessing you're using call_command here to make sure that the commands still work? If so, could you put a comment here (and elsewhere where you're doing this) explaining that this is what's going on?